commands/local_check_security.go

```diff
@@ -72,9 +72,22 @@ a specific "composer.lock" file.`,
 		terminal.Stdout.Write(output)
 
 		if os.Getenv("GITHUB_WORKSPACE") != "" {
+			gOutFile := os.Getenv("GITHUB_OUTPUT");
+
+            f, err := os.OpenFile(gOutFile, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
```

Comment:
Golang community uses tabs for indentation, you can use `go fmt` to automatically format the file.